### ISSUE TITLE
Is the saliencymodel for a specific classifier

### ISSUE THREAD

**Issue by yikekk** [PiotrDabkowski/pytorch-saliency#10]
Hi,
I wonder if the pre-trained saliency model for a specific classifier?
For example, how can i get the label of input.
It would be very thankful if you can open source the corresponding pre-trained classifier model.
